tests/test_morphology.cpp

```diff
@@ -20,3 +22,13 @@ TEST_CASE("LoadNeurolucidaMorphology", "[morphology]") {
 
     REQUIRE(m.diameters().size() == 14);
 }
+
+TEST_CASE("LoadBadDimensionMorphology", "[morphology]") {
+    REQUIRE_THROWS(morphio::Morphology("data/h5/v1/monodim.h5"));
+}
+
+TEST_CASE("LoadMergedMorphology", "[morphology]") {
+    auto file = HighFive::File("data/h5/merged.h5", HighFive::File::ReadOnly);
+    REQUIRE_NOTHROW(morphio::readers::h5::MorphologyHDF5(
```

Comment:
Can you test any property of the loaded morphology, just to make sure the morphology is not empty ?
You can test `rootSections.size()` for example.

Reply:
In order to test this I would need access to morphologies complete with the correct metadata, either in V1 or V2 versions that I would then manually merge. Do you know where I can find such morphologies? Thanks in advance!

Reply:
You can find some in `tests/data/h5/v1`